apps/sr-frontend/src/app/Character.tsx

```diff
@@ -67,12 +67,13 @@ export default function Character() {
                   {Object.values(relics).map((relic, index) => {
                     return (
                       <Grid item key={index} xs={1}>
-                        relic ? (
```

Comment:
The issue there is that you are printing `relic ? (` instead of using it as js logic. need to wrap `{}`